avo-hq/avo
# feature: add include_blank to select and country fields

# Description


Fixes https://github.com/avo-hq/avo/issues/1015

# Checklist:



## Manual review steps


1. go to a project
2. see that the country can be nullified
3. observer the null option is a string

1. go to a person
2. observe that the type can be nullified

Manual reviewer: please leave a comment with output from the test if that's the case.
